Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env groovy
+@Library('cht-jenkins-pipeline') _
+
+properties([
+    [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '15']],
```

Comment:
might as well add the weekly build property: `[pipelineTriggers([cron('H H(8-10) * * 4')])],` the final number is just the day of the week